test/scripts/ephe-domains-delete.sh

```diff
@@ -1,24 +1,12 @@
 #!/bin/bash
 
-function error {
-    local err=$?
-    printf "%s\n" "$1" >&2
-    exit $err
-}
-
-# make ephemeral-db-cli <<< "select domain_uuid from domains order by id desc limit 1;\\q"
+source "./test/scripts/ephe.inc.sh"
```

Comment:
I recommend that you do not rely on current working dir for source includes. Instead use this construct for all source includes:

```suggestion
source "$(dirname "${BASH_SOURCE[0]}")/common.inc.sh"
```

Reply:
I like this! thanks!